src/app/layout.tsx

```diff
@@ -1,3 +1,6 @@
+import { config } from '@fortawesome/fontawesome-svg-core';
+import '@fortawesome/fontawesome-svg-core/styles.css'; // Import the CSS
+config.autoAddCss = false; // Tell Font Awesome to skip adding the CSS automatically since it's being imported above
```

Comment:
@srkirkland yes, this is to fix that. it should work the same as before, nextjs just handles css differently and doesn't auto import the styles like it would normally. this is what the docs say to do, though there's more discussion in the github issue
https://docs.fontawesome.com/web/use-with/react/use-with 
https://github.com/FortAwesome/react-fontawesome/issues/134